server/meshmodel/helper.go

```diff
@@ -17,6 +17,8 @@ import (
 )
 
 var ArtifactHubComponentsHandler = meshmodel.ArtifactHub{} //The components generated in output directory will be handled by kubernetes
+var ModelsPath = "../meshmodel"
```

Comment:
This is different than this line, right? https://github.com/meshery/meshery/pull/8372/files#diff-76d1b6fe5a5ca5822c004882b83a214e655a9e59311faf7411411d6fc2f04787R63